src/core/tests/type_prop/istft.cpp

```diff
@@ -0,0 +1,427 @@
+        std::make_tuple(PartialShape{{47, 56}}, PartialShape{7}, 11, 3, false, PartialShape{6, {13, 16}, 2}),
+        std::make_tuple(PartialShape{{11, -1}}, PartialShape{7}, 11, 3, false, PartialShape{6, {1, -1}, 2}),
+        std::make_tuple(PartialShape{1, 16}, PartialShape{16}, 16, 16, false, PartialShape{1, 9, 1, 2}),
+        std::make_tuple(PartialShape{1, 48}, PartialShape{16}, 16, 16, false, PartialShape{1, 9, 3, 2}),
+        std::make_tuple(PartialShape{2, 48}, PartialShape{8}, 16, 4, false, PartialShape{2, 9, 9, 2}),
+        std::make_tuple(PartialShape{2, 9}, PartialShape{5}, 9, 100, false, PartialShape{2, 5, 1, 2}),
+        std::make_tuple(PartialShape{2, 0}, PartialShape{5}, 9, 100, true, PartialShape{2, 5, 1, 2}),
+        std::make_tuple(PartialShape{4, 47},
+                        PartialShape{7},
+                        11,
+                        3,
+                        false,
+                        PartialShape{4, 6, 13, 2}),  // If the signal was 48, the last sample can be lost
+        std::make_tuple(PartialShape{4, 56}, PartialShape{7}, 11, 3, false, PartialShape{4, 6, 16, 2}),
+        std::make_tuple(PartialShape{-1, {11, -1}}, PartialShape{7}, 11, 3, false, PartialShape{-1, 6, {1, -1}, 2}),
+        std::make_tuple(PartialShape{-1, {47, 56}}, PartialShape{7}, 11, 3, false, PartialShape{-1, 6, {13, 16}, 2}),
+        std::make_tuple(PartialShape{{2, 4}, {11, -1}},
+                        PartialShape{7},
+                        11,
+                        3,
+                        false,
+                        PartialShape{{2, 4}, 6, {1, -1}, 2}),
+        std::make_tuple(PartialShape{{2, 4}, {-1, -1}},
+                        PartialShape{7},
+                        11,
+                        3,
+                        true,
+                        PartialShape{{2, 4}, 6, {1, -1}, 2}),
+
+        std::make_tuple(PartialShape::dynamic(), PartialShape{7}, 11, 3, false, PartialShape::dynamic()),
+
+        std::make_tuple(PartialShape::dynamic(), PartialShape::dynamic(), 11, 3, false, PartialShape::dynamic()),
+        std::make_tuple(PartialShape::dynamic(), PartialShape::dynamic(), 11, 3, true, PartialShape::dynamic())),
+    testing::PrintToStringParamName());
+
+TEST_P(TypePropISTFTTestP, istft_shapes) {
+    const auto in_data = std::make_shared<Parameter>(element::f32, data_shape);
+    const auto window = std::make_shared<Parameter>(element::f32, window_shape);
+    const auto frame_size = Constant::create<int32_t>(element::i32, {}, {frame_size_val});
+    const auto frame_step = Constant::create<int32_t>(element::i32, {}, {step_size_val});
+
+    const auto op = make_op(in_data, window, frame_size, frame_step, center, normalized);
+    EXPECT_EQ(op->get_output_size(), 1);
+    EXPECT_EQ(op->get_output_element_type(0), element::f32);
+    EXPECT_EQ(op->get_output_partial_shape(0), signal_shape);
+}
+
+TEST_F(TypePropISTFTTest, istft_shape_length_const_out_1D) {
```

Comment:
```suggestion
TEST_F(TypePropISTFTTest, shape_length_const_out_1D) {
```
`istft` prefix from test can be removed as it is in test fixture.

Reply:
Done